MapboxMobileEvents/MMEConstants.h

```diff
@@ -135,13 +135,57 @@ extern NSString * const MMEEventUnknown;
 
 extern NSString * const MMEResponseKey;
 
-// SDK event source
+/*! @brief SDK event source */
 extern NSString * const MMEEventSource;
 
-// Log reporter HTML
-extern NSString * const MMELoggerHTML;
-extern NSString * const MMELoggerShareableHTML;
+#pragma mark - mobile.crash Keys
 
-@interface MMEConstants: NSObject
+extern NSString * const MMEEventMobileCrash;
+extern NSString * const MMEEventKeyOSVersion;
+extern NSString * const MMEEventKeyBuildType;
+extern NSString * const MMEEventKeyIsSilentCrash;
+extern NSString * const MMEEventKeyStackTrace;
+extern NSString * const MMEEventKeyStackTraceHash;
+extern NSString * const MMEEventKeyInstallationId;
+extern NSString * const MMEEventKeyThreadDetails;
+extern NSString * const MMEEventKeyAppId;
+extern NSString * const MMEEventKeyAppVersion;
+extern NSString * const MMEEventKeyAppStartDate;
+extern NSString * const MMEEventKeyCustomData;
+
+#pragma mark - MMEErrorDomain
+
+/*! @brief NSErrorDomain for MapboxMobileEvents */
```

Comment:
jazzy is more lenient than HeaderDoc about omitting `@brief`:

```suggestion
/// NSErrorDomain for MapboxMobileEvents
```

```suggestion
/*! NSErrorDomain for MapboxMobileEvents */
```